packages/frontend/README.md

```diff
@@ -1,68 +1,27 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# FlakyDashboard
```

Comment:
Just a general question, how do we handle style guidelines when it comes to writing our own code or reviewing a PR? Are there even style guidelines for angular skeleton code?

Reply:
Once the project is cloned and installed, there's a linter you can run to make sure that code is following conventions.

We will make sure we wire that up with the automated testing.